### PR TITLE
Increase certificates test

### LINKED ISSUE #10: Increase unit tests for certificates reconciler

### LINKED ISSUE #12: Increase unit tests for network package

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

Fix #10

# Changes

- :gift: More coverage for the certificates package

### CODE REVIEW

**Comment by matzew** [comment]
is there no code-coverage stats report from a bot ?

**Comment by slinkydeveloper** [comment]
@matzew I just noticed it now :smile: The coverage here is about 80%, uncovered code is just the `if err != nil` for errors coming from the std lib...

File: pkg/certificates/reconciler/certificates_test.go [hunk 1]
```diff
@@ -112,53 +112,82 @@ func TestReconcile(t *testing.T) {
...
+			wellFormedCaSecret.Name: func(t *testing.T, secret *corev1.Secret) {
 				require.Equal(t, wellFormedCaSecret, secret)
 			},
-			{
-				Namespace: namespace,
-				Name:      wellFormedDataPlaneSecret.Name,
-			}: func(secret *corev1.Secret) {
+			wellFormedDataPlaneSecret.Name: func(t *testing.T, secret *corev1.Secret) {
 				require.Equal(t, wellFormedDataPlaneSecret, secret)
 			},
```
Comment (matzew) [inline pkg/certificates/reconciler/certificates_test.go:142]: the callbacks for the `wellFormedDataPlaneSecret`/`wellFormedCaSecret` `.Name` seem very similar

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**